tests/client.rs

```diff
@@ -65,35 +65,32 @@ async fn has_collection() -> Result<()> {
     }
 }
 
-// #[tokio::test]
-// #[ignore]
-// async fn create_has_drop_collection() -> Result<()> {
-//     const URL: &str = "http://localhost:19530";
+#[tokio::test]
+async fn create_has_drop_collection() -> Result<()> {
+    const URL: &str = "http://localhost:19530";
+    const NAME: &str = "create_has_drop_collection";
```

Comment:
testing with random name is better

Reply:
i thought using a fixed collection name may be better for debugging and locating bugs